backend/tiggle-root/tiggle/src/main/kotlin/com/side/tiggle/global/auth/JwtTokenProvider.kt

```
@@ -1,94 +1,98 @@
+
+    private fun extractClaims(
+        jwtToken: String
+    ): Claims {
+        try {
+            return Jwts.parser()
+                .verifyWith(secretKey)
+                .build()
+                .parseSignedClaims(jwtToken)
+                .payload
+        } catch (e: JwtException) {
+            throw IllegalStateException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)